github/repos.go

```diff
@@ -201,6 +201,14 @@ type RepositoryListByOrgOptions struct {
 	// forks, sources, member. Default is "all".
 	Type string `url:"type,omitempty"`
 
+	// How to sort the repository list. Can be one of created, updated, pushed,
+	// full_name. Default: full_name
+	Sort string `url:"sort,omitempty"`
```

Comment:
It looks like all three of these parameters need to be `*string` instead of `string` since they are optional.

Note to self, this PR would mean a breaking API change and require a major semver version number bump.

Reply:
Wouldn't `omitempty` handle optional arguments @gmlewis ? And for what it's worth it seems like the other option structs in this file do not use `*string`

Reply:
Ah yes, sorry, you are right.  I'll strike my comment.  Thank you, @Xopherus!